pkg/spec/types.go

```diff
@@ -188,6 +194,11 @@ type JobSpecMod struct {
 	ActiveDeadlineSeconds *int64 `json:"activeDeadlineSeconds,conflicting,omitempty"`
 }
 
-type Controller struct {
-	Controller string `json:"controller,omitempty"`
+// TODOOOOOO!!
+// Ochestrator: OpenShift
+// DeploymentConfigSpecMod is Kedge's extension of OpenShift DeploymentConfig in order to define and allow
+// a complete kedge app based on OpenShift
+type DeploymentConfigSpecMod struct {
+	ControllerFields       `json:",inline"`
+	os_v1.DeploymentConfig `json:",inline"`
```

Comment:
Should this be `os_v1.DeploymentConfigSpec`?

Reply:
Good catch, you're right, similar to DeploymentSpec, it should be DeploymentConfigSpec for Openshift. https://github.com/kubernetes/kubernetes/blob/1a44e26670c9ce6b85304ff2603c462cb3a42bca/staging/src/k8s.io/api/apps/v1beta1/types.go#L284